### DISCUSSION TITLE
Website and Documentation Translations

### DISCUSSION THREAD

**Discussion by nazarepiedady** [strawberry-graphql/strawberry#2080] [Ideas]
Greetings everyone, I created this discussion to specifically discuss the availability of people
to start to translate to other speaking languages the website and also the documentation.

If there is someone that wants to help with this, let the community know about your interest.

So @patrick91 what do you think about this idea?

**Comment by patrick91** [comment]
this is a great idea! do you have experience in translating docs? 

I have no idea about what kind of tools we could use, our docs need a lot of attention at the moment, so we'll probably have a lot of updates in future, and we need to make sure we keep the translations in sync ☺️

**Reply by nazarepiedady** [reply]
Of course, I have.
I am the person who translates the [Nuxt.js Documentation](https://nuxtjs.org/pt/) to Portuguese.

The common way to serve different idioms of the same website is through routes, for example:

`https://strawberry.rocks/pt/` to serve the entire website and its dependency on Portuguese
`https://strawberry.rocks/es/` to serve the entire website and its dependency on Spanish

@patrick91

**Reply by patrick91** [reply]
Do you have anything that helps with making sure the docs are in sync between languages?

**Reply by nazarepiedady** [reply]
> Do you have anything that helps with making sure the docs are in sync between languages?

Are you saying about how to keep the different documentation equally updated?
@patrick91

**Reply by patrick91** [reply]
yes

**Reply by nazarepiedady** [reply]
@patrick91 in other projects I already participated we trust the people to
contribute with updates of the translations, because after having a first version of
translated document the update of them it is the more easy step to do.

So when do you think be a good idea to structure the project to support other
languages?

**Reply by patrick91** [reply]
@nazarepiedady do you want to jump on a call at some point, we can discuss this more in depth 😊 Email me, and we take it from there, if that's ok 😊

**Reply by nazarepiedady** [reply]
@patrick91 of course, but since I am working on the translation of the
documentation of many other technologies I will not have much time to
speak through a call, although we can share emails with more detail just
let me research the ways we can apply it to the existing website.
